landing-page/src/App.js

```diff
@@ -24,65 +25,61 @@ const styles = {
       margin: '16px',
       display: 'inline-block'
     },
-};
+});
 
-class App extends React.Component  {
+const App = () => {
 
-  state = {
-    userState : flowStates.SIGN_IN
-  }
+  const classes  = styles();
+  const [userState, setUserState] = useState(flowStates.AUTH_SUCCESS);
 
-  componentDidMount(){
+  useEffect(() => {
     document.body.style.backgroundColor = '#e0d9cd';
```

Comment:
why do you need to set style through `document.body`?

Reply:
Only way I know of to get a full background color for the entire page for now. As we go through styling iterations this may get removed eventually